docs/data/base/pages.ts

```diff
@@ -80,7 +80,7 @@ const pages = [
   },
   {
     pathname: '/base/guides',
-    title: 'How To Guides',
+    title: 'How to guides',
```

Comment:
```suggestion
    title: 'How-to guides',
```

(https://www.merriam-webster.com/dictionary/how-to, https://www.wikihow.com/%22How-to%22-or-How-to)